Formula/h/helidon.rb

```diff
@@ -35,6 +35,7 @@ def install
   end
 
   test do
+    mkdir_p HOMEBREW_CACHE/"java_cache"
```

Comment:
actually maybe a comment would be better context?

```suggestion
    # fix `HOMEBREW_CACHE/"java_cache"` does not exist issue
    mkdir_p HOMEBREW_CACHE/"java_cache"
```

Reply:
Addressed via https://github.com/Homebrew/homebrew-core/commit/6ea0f67a8cf22ce065f5128ab87c29d154c1147f.

Reply:
thanks! 👍